code/game/area/areas.dm

```diff
@@ -44,6 +44,7 @@
 	var/allow_construction = TRUE // whether or not you can build things like barricades in this area
 	var/is_landing_zone = FALSE // primarily used to prevent mortars from hitting this location
 	var/resin_construction_allowed = TRUE // Allow construction of resin walls, and other special
+	var/hangable = FALSE // Can a body be hung from this area
```

Comment:
don't super love having a variable on *every* area for this

Reply:
Agreed. Change this to an area flag or something, for environmental hanging